vars/parseStageInfo.groovy

```diff
@@ -112,5 +112,32 @@ def call(Map config = [:]) {
     if (config['ftest_arg']) {
       result['ftest_arg'] = config['ftest_arg']
     }
+
+    if (env.STAGE_NAME.contains('run_test.sh')) {
+      result['junit_files'] = 'test_results/*.xml'
+      result['artifacts'] = ['run_test.sh/*', 'vm_test/**']
+      result['with_valgrind'] = 'disabled'
+      result['valgrind_pattern'] = 'dnt.*.memcheck.xml'
+
+      if (config['junit_files']) {
+        result['junit_files'] = config['junit_files']
+      }
+      if (config['artifacts']) {
+        result['artifacts'] = config['artifacts']
+      }
+
+      if (config['with_valgrind']) {
+        result['with_valgrind'] = config['with_valgrind']
+      } else if(env.STAGE_NAME.contains('memcheck')) {
+        result['with_valgrind'] = 'memcheck'
+      }
+
+      if (result['with_valgrind'] == 'memcheck') {
+        result['valgrind_pattern'] = 'run_test_memcheck.sh/*memcheck.xml'
+        result['junit_files'] = ''
+        result['artifacts'] = ['run_test_memcheck.sh/*']
```

Comment:
Why is the memcheck version of the UT stage including this file as an artifact but the non-memcheck one is not?

Reply:
UT includes `run_test.sh` as an artifact, while UT with memcheck includes `run_test_memcheck.sh`. I decided a different name to identify the different logs.